site2/docs/client-libraries-cgo.md

```diff
@@ -4,7 +4,7 @@ title: Pulsar CGo client
 sidebar_label: "CGo(deprecated)"
 ---
 
-> The CGo client has been deprecated since version 2.7.0. If possible, use the [Go client](client-libraries-go) instead.
+> The CGo client has been deprecated since version 2.7.0. If possible, use the [Go client](client-libraries-go.md) instead.
```

Comment:
Can you request the client-libraries-go page after changes?

Reply:
Yes, without `.md` the page cannot be accessed.